### PR TITLE
fetch correct swap pools for each position token

### PR DESCRIPTION
**Pull request by azrield**
use:

USDC_TOKEN_TRACKER_ADDRESS_ETH = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"
USDC_TOKEN_TRACKER_ADDRESS_ARB = "0xaf88d065e77c8cc2239327c5edb3a432268e5831"

in .env for USDC trackers on our supported chains

see env.example for updated .env fields (removed many unused)

### CODE REVIEW

File: src/analyzeRoutine.js [hunk 1]
```diff
@@ -10,40 +10,42 @@ const ETHEREUM_CHAIN_ID = 1;
...
+        : await getPoolExchangeRate(position, 0);
+
+    const Token1USDCRate =
+      token1Symbol === "USDC" || token1Symbol === "USDT"
+        ? 1
+        : await getPoolExchangeRate(position, 1);
+
     const currentBlockNumber = await getCurrentBlockNumber(position.chain);
 
     savePositionDataSQL(
```
Comment (yuvals2146) [inline src/analyzeRoutine.js:30]: renamed already but leave it I will fixed it in the next pr

File: src/blockchain/getPostionData.js [hunk 2]
```diff
@@ -354,7 +391,7 @@ const getPoolExchangeRate = async (poolAddress, chain) => {
         : provider._network.name;
 
     throw new Error(
-      `error with retrive pool exchange rate for pool ${poolAddress} on provider ${providerName}`
+      `Error while retrieving pool exchange rate for pool ${poolAddress} on provider ${providerName}`
```
Comment (yuvals2146) [inline src/blockchain/getPostionData.js:394]: melech

File: src/engine/analyzer.js [hunk 3]
```diff
@@ -29,9 +29,19 @@ async function analyzeDataPoint(
...
+      "Left:",
+      positionData.tickLeft,
+      "Right:",
+      positionData.tickRight,
+      "Current:",
+      positionData.tickCurr
+    );
     await notify(
-      `Position Out of Limits: ${positionData.tickCurr}`,
+      `Position ${positionId} is out of limits: Left: ${positionData.tickLeft}, Right: ${positionData.tickRight}, Curr: ${positionData.tickCurr}`,
```
Comment (yuvals2146) [inline src/engine/analyzer.js:44]: need to changed to position object ({id: x, chain: y})

Comment (yuvals2146) [reply src/engine/analyzer.js:44]: as for this is the convention in our code

**Review by yuvals2146 — COMMENTED** [review]
Looks great, I didn't verify any of the logic and math but if the numbers make sense it LGTM 👑

File: src/analyzeRoutine.js (hunk 1 above)
Comment (yuvals2146) [reply src/analyzeRoutine.js:30]: talking about `savePositionDataSQL`

**Comment by yuvals2146** [comment]
BTW, didn't you intended to add it to the db?

File: src/engine/analyzer.js (hunk 3 above)
Comment (azrield) [reply src/engine/analyzer.js:44]: this is actually only the id of the positioon (no chain)

**Review by yuvals2146 — APPROVED** [review]

**Pull request merged**